subsys/bluetooth/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+add_library(subsys__bluetooth INTERFACE)
```

Comment:
Thanks for this change. It makes sense as a way to have private include directories.

Do you think this pattern can be replaced by a function or macro that does string manipulation on CMAKE_CURRENT_SOURCE_DIR?